server/repository/migrations/postgres/2023-04-21_T10-00_changes_made_in_develop/down.sql

```diff
@@ -0,0 +1,81 @@
+-- This file should undo anything in `up.sql`
+
+-- 2021-08-15T10-00_create_name_table
+CREATE TYPE gender_type_original AS ENUM (
```

Comment:
There's no convenient way to remove values from an existing Enum. Hence this whole routine 🙄